main.nf

```diff
@@ -81,10 +81,10 @@ if (params.skip_filter_ref) {
     reads_ch = Channel.empty()
 }
 
-process filterRefReads {
+process prefilterHostReads {
```

Comment:
Could you also change the `overview.md` and `output.md` documents to reflect the process, flag, and output name changes?

Reply:
Updated the PR to add changes to `output.md`. I had previously updated `overview.md`, maybe I missed something, but didn't notice anything when I checked again.